internal/workflow/activities/bundle.go

```diff
@@ -101,51 +90,28 @@ func (a *BundleActivity) Execute(ctx context.Context, params *BundleActivityPara
 
 	err = unbag(res.FullPath)
 	if err != nil {
-		return nil, temporal_tools.NewNonRetryableError(err)
-	}
-
-	res.RelPath, err = filepath.Rel(params.TransferDir, res.FullPath)
-	if err != nil {
-		return nil, temporal_tools.NewNonRetryableError(fmt.Errorf(
-			"error calculating relative path to transfer (base=%q, target=%q): %v",
-			params.TransferDir, res.FullPath, err,
-		))
+		return nil, temporal_tools.NewNonRetryableError(
+			fmt.Errorf("bundle: unbag: %v", err),
+		)
 	}
 
 	if err = setPermissions(res.FullPath); err != nil {
```

Comment:
Should this use the new `fsutil.SetFileModes` function?

Reply:
Yes, totally :+1: